src/core/LiterallyCanvas.coffee

```diff
@@ -313,7 +320,7 @@ module.exports = class LiterallyCanvas
     return unless shapes.length
     drawShapes = =>
       for shape in shapes
-        renderShapeToContext(ctx, shape, {retryCallback})
+        shape.draw(ctx, { retryCallback })
```

Comment:
Yep! That was why I used shape.draw instead of renderShapeToContext! I guess I had the direction of that backwards :)